question26/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="ja">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+        <link href="./css/style.css" rel="stylesheet" />
+        <script type="module" src="./js/login.js"></script>
+        <title>TerraceTechフロントエンドエンジニア養成所</title>
+    </head>
+    <body>
+        <div id="js-wrapper" class="wrapper">
+            <h1>Login</h1>
+            <form method="post" id="form" novalidate>
+                <div>
+                    <ul>
+                        <li>
+                            <label for="name_mail">ユーザー名/メールアドレス</label>
+                            <input type="text" id="name_mail" name="name_mail" required class="input"/>
+                            <span class="error"></span>
+                        </li>
+                        <li>
+                            <label for="password">パスワード</label>
+                            <input type="password" id="password" name="user_password" required class="input"/>
+                            <span class="error"></span>
+                        </li>
+                    </ul>
+                </div>
+                <button type="button" id="js-submit-button" class="submit_button" disabled>送信</button>
+            </form>
+            <div class="link">
+                <div>
+                    <a href="./forgotPassword.html">パスワードを忘れた方はこちら</a>
+                </div>
+                <div>
+                    <a href="./register.html">会員登録がまだの方はこちら</a>
+                </div>
```

Comment:
nit
I don't think `<div>` is needed to modify the scope of the link.
This is because` <a>` with `display:inline-block` will solve the problem.
```suggestion
                <a href="./forgotPassword.html">パスワードを忘れた方はこちら</a>
                <a href="./register.html">会員登録がまだの方はこちら</a>
```

Reply:
I also considered inline-block, but Upon further inspection, it appears that inline-block is to align elements horizontally. At first glance, they appear to be aligned vertically, but when the screen size is changed, they are unstable, so their use was determined to be inappropriate.


https://user-images.githubusercontent.com/51282804/194298845-3dade5e9-6b35-450e-a3cf-437a4961f418.mov

Reply:
Oh... sorry...🙏